diff --git a/src/productcatalogservice/product_catalog.go b/src/productcatalogservice/product_catalog.go
--- a/src/productcatalogservice/product_catalog.go
+++ b/src/productcatalogservice/product_catalog.go
@@ -62,8 +62,7 @@ func (p *productCatalog) SearchProducts(ctx context.Context, req *pb.SearchProdu
 	time.Sleep(extraLatency)

 	var ps []*pb.Product
 	for _, product := range p.parseCatalog() {
-		if strings.Contains(strings.ToLower(product.Name), strings.ToLower(req.Query)) ||
-			strings.Contains(strings.ToLower(product.Description), strings.ToLower(req.Query)) {
+		if strings.Contains(strings.ToLower(product.Name), strings.ToLower(req.Query)) {
 			ps = append(ps, product)
 		}
 	}
