diff --git a/src/currencyservice/server.js b/src/currencyservice/server.js
--- a/src/currencyservice/server.js
+++ b/src/currencyservice/server.js
@@ -150,8 +150,8 @@ function convert (call, callback) {
       euros.nanos = Math.round(euros.nanos);

       // Convert: EUR --> to_currency
       const result = _carry({
-        units: euros.units * data[request.to_code],
-        nanos: euros.nanos * data[request.to_code]
+        units: euros.units / data[request.to_code],
+        nanos: euros.nanos / data[request.to_code]
       });

       result.units = Math.floor(result.units);
